README.md

```diff
@@ -19,37 +10,39 @@
   - [💻 Prerequisites](#prerequisites)
   - [💻 Setup](#setup)
   - [👤**Authors** ](#authors-)
+  - [🔭 Future Features](#future-features)
   - [🤝 Contributing ](#-contributing-)
   - [⭐️ Show your support ](#️-show-your-support-)
   - [🙏 Acknowledgments ](#-acknowledgments-)
   - [📝 License ](#-license-)
 
-# 🎥 Movie Web app<a name="about-project"></a>
-
-*tittle* is a web app that allows you to ... 
-
-Here are some features of the page:
-
-Add your likes and comments on your web app.
+# 🎥 Moanfi Restaurant <a name="about-project"></a>
 
+MoanFi restaurant is an app that let users chose a meal form its rich and diverse
+gastronomical selection of meals. It allows users to chose and get details on meal region and originated area. As well as oppurtunitu to reserve for a meal at a particular date and see which meals ahve been reserved.
 
 ## 🛠 Built With <a name="built-with"></a>
+
     HTML
     CSS
     Javascript
     Webpack
 
+### Key Features <a name="key-features"></a>
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+- **Add Reservation**
+- **Add Comment**
+- **Like a Meal**
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-none
+You would be able see later a Live Version of this App Below
+ https://fitz95.github.io/API-BASED-APP-/
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-
 ## 💻 Getting Started <a name="getting-started"></a>
```

Comment:
- [ ] OPTIONAL
Kindly include appropriate instructions under the _Getting Started_ section, especially on how to _run_ and _test_ your app,

Reply:
done